application/extensions/SettingsWidget/SettingsWidget.php

```diff
@@ -612,33 +612,56 @@ public function renderDate($name, array $metaData, $form = null)
     {
         $dateformatdetails = getDateFormatData(Yii::app()->session['dateformat']);
         $value = isset($metaData['current']) ? $metaData['current'] : '';
-        $html = Yii::app()->getController()->widget('yiiwheels.widgets.datetimepicker.WhDateTimePicker', array(
-                'name' => $name,
-                'id' => \CHtml::getIdByName($name),
+        $inputName = $name;
+
+        /* Optionnal save format in setting option, if not set use default */
+        if (!isset($metaData['saveformat'])) {
+            $metaData['saveformat'] = "Y-m-d H:i";
```

Comment:
Wouldn't this change the current default?

Reply:
> You think i must remove this commit https://github.com/LimeSurvey/LimeSurvey/commit/aacf731f18556d52a7a637f6d4eb7242cb3be220 ? I don't know, maybe … unsure ?

Well, seems this is not backward compatible.
The whole idea was that this enhancement was optional and hence bacward compatible.
Is it?